Core/Core/Pages/PageEditor/PageEditorView.swift

```diff
@@ -76,25 +81,24 @@ public struct PageEditorView: View {
     var form: some View {
         ScrollView { VStack(alignment: .leading, spacing: 0) {
             if env.app == .teacher {
-                Text("Title")
+                Text("Title", bundle: .core)
                     .font(.semibold14).foregroundColor(.textDark)
                     .padding(EdgeInsets(top: 16, leading: 16, bottom: 8, trailing: 16))
                 Divider()
                 TextField("Add Title", text: $title)
```

Comment:
This needs localization too

Reply:
I couldn't find a constructor for `TextField` that took `Text` or `() -> Label`.

Reply:
should I just use NSLocalizedString then? There's a constructor that takes a LocalizedStringKey, but you can't construct one of those with a bundle.

Reply:
Weird, that does seem to be missing. NSLocalizedString then?